assets/js/span-button.js

```diff
@@ -0,0 +1,13 @@
+function addSpanEventHandlers() {
+  const spanElements = document.querySelectorAll('span[role="button"]');
```

Comment:
Totally a nit for this project, but historically the best practice for this sort of DOM-augmenting JS plugin is for it to be totally agnostic to what elements it wraps. Typically the function accepts a collection of HTML elements as an argument, and app.js queries the DOM and passes the result to the plugin.

Reply:
No longer relevant with the custom element rewrite